src/test/regress/expected/subquery_local_tables.out

```diff
@@ -242,8 +242,20 @@ DEBUG:  Plan XXX query after replacing subqueries and CTEs: SELECT user_id FROM
 (2 rows)
 
 SET client_min_messages TO DEFAULT;
+-- Test https://github.com/citusdata/citus/issues/2717
+create table test_dist (id int, table_name text, column_name text);
+select create_distributed_table('test_dist','id');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+select * from test_dist td join information_schema.columns icl on icl.table_name::text = lower(td.table_name) and icl.column_name::text = lower(td.column_name);
+ id | table_name | column_name | table_catalog | table_schema | table_name | column_name | ordinal_position | column_default | is_nullable | data_type | character_maximum_length | character_octet_length | numeric_precision | numeric_precision_radix | numeric_scale | datetime_precision | interval_type | interval_precision | character_set_catalog | character_set_schema | character_set_name | collation_catalog | collation_schema | collation_name | domain_catalog | domain_schema | domain_name | udt_catalog | udt_schema | udt_name | scope_catalog | scope_schema | scope_name | maximum_cardinality | dtd_identifier | is_self_referencing | is_identity | identity_generation | identity_start | identity_increment | identity_maximum | identity_minimum | identity_cycle | is_generated | generation_expression | is_updatable
```

Comment:
Would be nice to simplify the output (e.g. in case this changes in future PG versions) and may have some actual rows to confirm the query does the right thing.